pom.xml

```diff
@@ -1269,6 +1270,25 @@
                     io.dockstore.common.BenchmarkTest,io.dockstore.common.ToilOnlyTest, io.dockstore.common.NonConfidentialTest
                 </excludeGroups>
             </properties>
+
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>com.github.spotbugs</groupId>
+                        <artifactId>spotbugs-maven-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <goals>
+                                    <goal>check</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                        <configuration>
+                            <skip>false</skip>
```

Comment:
Is the reason you have it in two profiles instead of just one because this profile is only run locally, and not on the CI servers?

Reply:
I think the CI servers always use a profile, so I had to add it to the unit-tests profile to have to have it run somewhere during CI. 

I also added it to the default activation so we get it locally when doing normal builds like `mvn clean install`